apps/frontend/server/api/universes.get.ts

```diff
@@ -0,0 +1,13 @@
+import { getServerSession } from '#auth';
+import { Client } from '@skyra/internal';
+
+export default eventHandler(async (event) => {
+	const session = await getServerSession(event);
+	if (!session?.user) return [];
+
+	console.log({ session });
+	// TODO: next-auth is dumb and does not support `.id`. Refs:
```

Comment:
Is there an open issue on next-auth for this? If not, make one. Don't make the problem worse by not alerting the maintainers.